.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+<!-- 
+	Please use the following format when naming your PR
+	< Issue Number >:< Issue Description >
+	e.g. KEP-000: adding beta graduation criteria
+	
+	Avoid using phrases like `fixes #NNNN` in the description
+	unless the pull request is to change the KEP status to 
+	implemented or KEP has been deprecated.
+-->
+
+<!-- short description of work done in PR e.g. updating milestone, adding new KEP, adding test requirements… -->  
+- One-line PR description:
+
+<!-- link to the k/enhancements issue -->
+- Issue link:
```

Comment:
Wondering if we want some optional subsection (at the end) for general comments or extra info, since many times authors want to add information longer than 1 line?  Scrolling through PRs in repo confirms that many authors do add some extra info to give more detail/context. Would make sense to add a labeled section for that since they probably will add it anyway? 

Wdyt @annajung

Reply:
that makes sense, added another section for comments